services/explorer/src/components/BlockTable/BlockTable.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import {
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  Box,
+  TablePagination,
+  TableContainer,
+  ThemeProvider,
+  CssBaseline,
+} from '@mui/material';
+import { Label, Button, Icons } from '@taraxa_project/taraxa-ui';
+import type { TransactionData } from '../../models/TransactionData';
+import { TransactionStatus } from '../../models/TableData';
+import { theme } from '../../theme-provider';
+import { AddressLink } from '../Links';
+
+export interface TransactionDataItem extends TransactionData {
+  txHash: string;
+  dagLevel: string;
+  dagHash: string;
+}
+
+export const statusToLabel = (state: TransactionStatus) => {
```

Comment:
I don't think you need this hwere. You can reuse it from utils/TransactionRow.ts